### PR TITLE
Feat:added linter 

### PR DESCRIPTION
**Pull request by omjikush09**
Hi, added the linter.  This is the first time, I am using PHP. So I may have not installed all dependencies correctly or removed some dependencies. So Please review it. And suggest changes...

### CODE REVIEW

**Comment by omjikush09** [comment]
As Travis CI is failing. So Do I need to remove some packages?

**Comment by omjikush09** [comment]
Issue https://github.com/appwrite/appwrite/issues/4362

File: composer.json [hunk 1]
```diff
@@ -28,11 +28,12 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.3",
-        "squizlabs/php_codesniffer": "^3.6"
+        "squizlabs/php_codesniffer": "^3.6",
```
Comment (Meldiron) [inline composer.json:31]: Note for @christyjacob4 , can we now remove this as we are not using phpcbf/phpcs anymore?

**Review by Meldiron — APPROVED** [review]
Looks great 🎉

Since this is deprecating existing linter, I would prefer to have one more PR review before marking this finished. @christyjacob4 can you please? 🙏 I would love your review on `travis.yml`, `linter.yml`, `composer.json`, `CONTRIBUTING.md`, `Dockerfile` and `phpcs.xml`.

File: .github/workflows/linter.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        fetch-depth: 2
+
+    - run: git checkout HEAD^2
+
+    - name: Run Linter
+      run: |
+        docker run --rm -v $PWD:/app composer sh -c \
+        "composer install --profile --ignore-platform-reqs && composer lint"
+
+
```
Comment (christyjacob4) [inline .github/workflows/linter.yml:22]: Let's remove the extra lines

File: composer.json [hunk 3]
```diff
@@ -28,11 +28,12 @@
...
+        "laravel/pint": "^1.2"
```
Comment (christyjacob4) [inline composer.json:32]: @Meldiron did we agree on which version to use ? We might want to keep it consistent across all the PRs

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Looks good 👍  just some final comments

File: composer.json (hunk 3 above)
Comment (Meldiron) [reply composer.json:32]: We used `dev-main` in example PR, that's what other contributors are using. Let's use `dev-main` here too for `pint`.

File: .github/workflows/linter.yml (hunk 2 above)
Comment (omjikush09) [reply .github/workflows/linter.yml]: Done

File: composer.json (hunk 3 above)
Comment (omjikush09) [reply composer.json]: Done!

**Comment by omjikush09** [comment]
@christyjacob4,  Made the changes as requested. 😁

**Review by Meldiron — APPROVED** [review]
Looks great 🎉 I made one commit myself to address some tiny changes quickly.

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**